### ISSUE TITLE
ÅÄÖåäö looks way off following recent update.

### ISSUE THREAD

**Issue by smlsvnssn** [CatharsisFonts/Cormorant#30]
The kerning is wrong, and the rings and umlauts are off center.
![image](https://cloud.githubusercontent.com/assets/3297282/22270076/56d6c50e-e28e-11e6-84f7-076552921cb5.png)

**Comment by CatharsisFonts** [comment]
Dear Samuel,

Thanks for the heads-up. This is a known issue that has been fixed in the current live version on GitHub, and is queued to be released on Google Fonts as soon as possible. I'll make a new Release soon as well. We ran afoul of a temporary bug in the Glyphs software. If you're interested in the backstory, here it is:

https://groups.google.com/d/msg/googlefonts-discuss/wmHBh2k_eu0/F3iPOm0tCQAJ

Cheers, Christian
